### PR TITLE
Wda3-158: Leerroute opslaan & wijzigen

### PR DESCRIPTION
**Pull request by tromphakvoort**

### CODE REVIEW

File: src/Components/Leerroute/StudyRouteYear.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      <div class="p-8">
+        <div class="flex justify-between items-center mb-4">
+          <h2 class="text-xl font-semibold">Module keuze</h2>
+          <div class="ml-auto flex">
+            <button id="closeModal" class="bg-gray-300 hover:bg-gray-400 text-gray-800 font-bold py-2 px-4 rounded-full focus:outline-none focus:shadow-outline">Sluit</button>
+          </div>
+        </div>
+      </div>
+      <div class="flex flex-wrap -mx-4 justify-evenly" id="blockContainer">
+        <!-- Dynamic blocks will be added here -->
```
Comment (s1182276) [inline src/Components/Leerroute/StudyRouteYear.js:42]: BlockcontainerInfo div is missing here, causing a regression on the ModuleInfo functionality. It has been lost by refactoring from Year.js

**Review by s1182276 — APPROVED** [review]
Tested and discussed regressions to be solved in Teams.

File: src/Components/Leerroute/StudyRouteYear.js (hunk 1 above)
Comment (tromphakvoort) [reply src/Components/Leerroute/StudyRouteYear.js:42]: Thanks for the feedback! Issue got resolved in https://github.com/ICTDTWDa3/Client/pull/26/commits/c4abd652061da2dfe849cf18e85acd1d78e68ab6

Comment (s1182276) [reply src/Components/Leerroute/StudyRouteYear.js:42]: Fixed in https://github.com/ICTDTWDa3/Client/pull/26/commits/c4abd652061da2dfe849cf18e85acd1d78e68ab6

**Pull request merged**